packages/blocks/src/database-block/table/table-view.ts

```diff
@@ -54,6 +54,17 @@ const styles = css`
     z-index: 1;
     overflow-x: scroll;
     overflow-y: hidden;
+    /* border-top: 1.5px solid var(--affine-border-color); */
+    padding-top: 1.5px;
+  }
+  .affine-database-block-table::before {
+    content: '';
+    position: absolute;
+    left: 0;
+    right: 0;
+    top: 0;
+    height: 1.5px;
+    background-color: var(--affine-border-color);
```

Comment:
@zanwei Is this as expected?

Reply:
1.5px border at top of table view

Reply:
Based on what we discussed earlier, it should be 1px border at top of table view

Reply:
Got it!